_d/monetization.md

```diff
@@ -246,4 +248,6 @@ Most successful creators combine:
 
 The real money isn't in any single platform - it's in building a brand that transcends platforms. Create because you love it, and treat any money as a bonus.
 
-For more on content creation strategies, see our [content creation guide](/content-creation). For deeper advertising insights, check out our [advertising fundamentals](/advertising).
+{% include summarize-page.html src="/content-creation" %}
+
+{% include summarize-page.html src="/advertising" %}
```

Comment:
No put these where you are putting think links not just as the end. Make them more natural.